src/app.hooks.js

```diff
@@ -5,10 +5,11 @@ const sanitize = (data, options) => {
 	// https://www.npmjs.com/package/sanitize-html
 	if ((options||{}).html === true) {
 		// editor-content data
+		// TODO what is 'rechnen' used for?
 		data = sanitizeHtml(data, {
 			allowedTags: [ 'h1', 'h2', 'h3', 'blockquote', 'p', 'a', 'ul', 'ol', 's', 'u', 'span', 'del',
 				'li', 'b', 'i', 'img', 'strong', 'em', 'strike', 'code', 'hr', 'br', 'div', 'rechnen',
-				'table', 'thead', 'caption', 'tbody', 'tr', 'th', 'td', 'pre', 'audio', 'video' ],
+				'table', 'thead', 'caption', 'tbody', 'tr', 'th', 'td', 'pre', 'audio', 'video', 'iframe' ],
```

Comment:
isn't this an invitation for all kinds of shady stuff? @janrenz @adrianjost

Reply:
yes it is, but we don't really have another option. 
We can choose between: 
1. allow iFrames and users can embed videos
2. remove iFrames and users can't include videos
The editor we are using is often using an iFrame for videos.